README.md

```diff
@@ -54,7 +54,19 @@ $ python setup.py build_ui
 ```
 Note that if you make changes to the UI using the `.ui` forms, you must re-build using the same command.
 
+#### Cython Files:
+to speed up some more computationally expensive parts of the codebase, openMotor uses the cython programming language to compile the code to C.
+
+Because the cyphon code must be compiled separately for each system, you must build them by running:
+```
+$ python setup.py build_ext --inplace
+```
+Note that if you make changes to any of the `.pyx` files, you must re-compile the library using the same command.
+
+#### Final Compilation Step:
```

Comment:
Nitpick: Python is an interpreted language, so "compilation" isn't quite right. Maybe `Run the Application` or something like that?